src/api/streams/updateStream.js

```diff
@@ -13,9 +13,13 @@ import { apiPatch } from '../apiFetch';
 export default (
   auth: Auth,
   id: number,
-  property: string,
-  value: string | boolean,
-): Promise<ApiResponse> =>
-  apiPatch(auth, `streams/${id}`, {
-    [property]: value,
-  });
+  params: $ReadOnly<{|
+    description?: string,
+    new_name?: string,
+    is_private?: boolean,
+    is_web_public?: boolean,
+    stream_post_policy?: number,
```

Comment:
Question: Should we try for parity of the Zulip APIs or leave out properties that aren't used in the mobile app yet?

Reply:
Good question. Chat thread: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/api.2EupdateStream/near/1347906.2E01